clients/mongoStoreClient.go

```diff
@@ -118,8 +119,8 @@ func constructQuery(details *model.QueryData) (query bson.M, sort string) {
 	for _, v := range details.MetaQuery {
 		log.Println("constructQuery: create base queries")
 		//base query
-		queryThis := bson.M{"groupId": v}
-		queryThat := bson.M{"_groupId": v, "_active": true}
+		queryThat := bson.M{"groupId": v}
+		queryThis := bson.M{"_groupId": v, "_active": true}
```

Comment:
What are the implications of the fact that `"_active": true` wasn't there before? Any?